### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by KAMiKAZOW**
No CLA, just GPLv2+ (or compatible) for new code.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,112 +1,3 @@
...
-[aseprite-discuss](http://groups.google.com/group/aseprite-discuss)
-group. If you want to start working in something
-(e.g. [an open issue](https://github.com/aseprite/aseprite/issues)),
-you should send an email to the group describing what you want to do,
-to avoid programming in something that is already done or which
-someone else is working on.
-
-And always remember to take a look at our
-[roadmap](http://www.aseprite.org/roadmap/).
+Newly contributed code must be licensed under [GNU GPL 2.0](https://www.gnu.org/licenses/gpl-2.0.html) with “or any later version” clause or a [compatible license](https://www.gnu.org/licenses/license-list.html#GPLCompatibleLicenses).
```
Comment (akien-mga) [inline CONTRIBUTING.md]: The current license is GPLv2 without the "or any later version" clause. Do we want to use a different license from now on (i.e. GPLv2+ as you suggest), or should we keep the plain GPLv2?

Comment (dacap) [inline CONTRIBUTING.md]: As @akien-mga said, Aseprite was distributed under GPLv2 (no GPLv2+ or v3). I don't remember when I've made the change from GPLv2+ to GPLv2 (the reasons were just "let's wait to understand GPLv3 and then see if it worth it").

**Comment by KAMiKAZOW** [comment]
I followed the example set by LibreOffice. They forked LGPLv3-only OpenOffice but decided to accept new code only with "or any later version" clause (or a compatible license).

**Comment by Conan-Kudo** [comment]
I would suggest going with GPLv2+ for new code, so that as code churns, the license can evolve, eventually.

**Pull request merged**